scripts/run_docker_build.sh

```diff
@@ -16,6 +16,7 @@ conda-build:
 
 always_yes: true
 show_channel_urls: true
+ssl_verify: false
```

Comment:
We have a firewall using their own certificate.  This is a pain for every software we use.